src/Changelogs/ChangelogDatastore.ts

```diff
@@ -82,7 +82,28 @@ export class ChangelogDataStore {
         { $group: { _id: '$learningObjectId', logs: { $push: '$logs' } } },
       ])
       .toArray();
-    return changelogs;
+  }
+
+  /**
+   * Get all change logs for the specified Learning Object that were created before the given date
+   *
+   * @param {string} learningObjectId The id of the learning object that the requested changelog belongs to
+   * @param {string} date The date of that the changelog was created
+   */
+  async fetchChangelogsBeforeDate(params: {
+    learningObjectId: string,
+    date: string,
+  }): Promise<ChangeLogDocument[]> {
+    return await this.db
```

Comment:
`async await` not needed here if we are just returning the Promise.

```suggestion
    return  this.db
```